pkg/operator/petset.go

```diff
@@ -208,6 +219,12 @@ func makePetSetSpec(name, image, version, retention string, replicas int32, aler
 								MountPath: "/etc/prometheus/config",
 							},
 						},
+						Resources: v1.ResourceRequirements{
```

Comment:
I'll keep an eye on the mem usage, regarding cpuquota this should be fine though